src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx

```diff
@@ -0,0 +1,51 @@
+---
+subject: Docs
+releaseDate: '2024-04-12'
+version: '4-12-2024'
+---
+
+<Callout variant="important">
+We're trying out release notes to let you know what's changing in our docs each week. This is an experiment, so [let us know via GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose) if you find them helpful!
+</Callout>
+
+### New docs
+
+* General availability for browser [Session replay](/docs/browser/browser-monitoring/browser-pro-features/session-replay/), previously in limited preview. Check it out for deeper insights into user interactions on your websites.
+* Docs on our new [Unreal Engine plugin](/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application/) to monitor Unreal Engine mobile apps, and updates to mobile agent API docs with Unreal Engine plugin methods.
+* New Prometheus integration for [Prometheus Node Exporter](/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration/), which lets you monitor Linux hardware and kernel metrics.
+
+### Major changes
+
+* Continued rewrite of on-host integration docs to make it easier to install and ensure the docs are accurate and up to date. Up this week: [Sonarqube](/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration) and [Windows services](/docs/infrastructure/host-integrations/host-integrations-list/windows-services-integration).
+* Added info on new mobile user journeys feature to [Crash analysis # User journeys](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/crash-analysis-page/#user-journeys).
```

Comment:
Do you really want that "#" between crash analysis and user journeys in the link text?

Reply:
I do but no one else seems to like it. :D